.github/workflows/test.yaml

```diff
@@ -40,7 +40,7 @@ jobs:
       - name: Download k3d
         run: wget -q -O - https://raw.githubusercontent.com/k3d-io/k3d/main/install.sh | bash
       - name: Create k3d cluster
-        run: k3d cluster create --agents 2 -p "80:80@loadbalancer" --k3s-arg "--disable=traefik@server:0" --registry-create reciperegistry:51351
+        run: k3d cluster create --agents 2 -p "80:80@loadbalancer" --k3s-arg "--disable=traefik@server:*" --k3s-arg "--disable=servicelb@server:*" --registry-create reciperegistry:51351
```

Comment:
Could you please add a comment here on why this is disabled? Just for context for anyone looking at this code in the future.

Reply:
@kachawla The internal load balancer in k3d conflicts with the routing provided by Radius. The issue was originally reported in the Radius repo, but it also applies here: https://github.com/radius-project/radius/issues/7637.

Reply:
Thanks @brooke-hamilton. I meant adding a code comment in the workflow :)

Reply:
Done! My mistake - your comment was clear and applied to the code. 💯